srml/council/src/lib.rs

```diff
@@ -14,7 +14,221 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Council system: Handles the voting in and maintenance of council members.
+//! # Council Module
+//!
+//! The Council module provides tools to manage the council and proposals. The main components are:
+//!
+//! - **Council Seats:** Election of councillors.
+//! 	- [`seats::Trait`](./seats/trait.Trait.html)
+//! 	- [`Call`](./seats/enum.Call.html)
+//! 	- [`Module`](./seats/struct.Module.html)
+//! - **Council Motions:** Voting as a body to dispatch calls from the special `Council` origin.
+//! 	- [`motions::Trait`](./motions/trait.Trait.html)
+//! 	- [`Call`](./motions/enum.Call.html)
+//! 	- [`Module`](./motions/struct.Module.html)
+//! - **Council Voting:** Proposals sent to [`Democracy Module`](../srml_democracy/index.html) for public referenda.
+//! 	- [`voting::Trait`](./voting/trait.Trait.html)
+//! 	- [`Call`](./voting/enum.Call.html)
+//! 	- [`Module`](./voting/struct.Module.html)
+//!
+//! ## Overview
+//!
+//! The Council module provides functionality to handle:
+//!
+//! - The voting in and maintenance of council members.
+//! - Proposing, vetoing, and passing motions.
+//!
+//! The council is an on-chain entity comprised of a set of account IDs, with the role of representing
+//! passive stakeholders. Its primary tasks are to propose sensible referenda and thwart any uncontroversially
+//! dangerous or malicious referenda.
+//!
+//! ### Terminology details
+//!
+//! #### Council Motions (motions.rs)
+//!
+//! Internal proposals that are only proposed and voted upon by _councillors_.
+//! Each proposal has a minimum threshold of yay votes that it needs to gain to be enacted.
+//!
+//! - **Council motion:** A mechanism used to enact a proposal.
+//! - **Proposal:** A submission by a councillor. An initial vote of yay from that councillor is applied.
+//! - **Vote:** A vote of yay or nay from a councillor on a single proposal. Councillors may change their vote but a
+//!   duplicate vote will return an error.
+//!
+//! Upon each vote, if the threshold is reached, the proposal is is executed. Similarly,
+//! if the number of nay votes is high enough such that it could not pass even if all other councillors
+//! (including those who have not voted) voted yay, the proposal is dropped.
+//!
+//! Note that a council motion has a special origin type, [`seats::Origin`](./motions/enum.Origin.html), that limits
+//! which calls can be dispatched. The [Treasury](../srml_treasury/index.html) module is an example that uses the
+//! council motion origin.
```

Comment:
```suggestion
//! that uses the council motion origin.
```

Reply:
it looks like the suggested words are already on the previous line